.github/workflows/all.yml

```diff
@@ -83,6 +83,28 @@ jobs:
           fi
           go test "${args[@]}" -timeout 800s ./...
 
+  tc39:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Do we want to run this on every commit? We might be OK with just running them on each release (version tag), especially if they're flaky, and we should probably exclude it from the requirements for a successful build.

Reply:
1. there are currently no requirements (I think)
2. they aren't really flaky ... just they sometimes take too much time - predomenantly when I increase them or decide to try to enable parallel execution ... OR maybe they take too much memory :man_shrugging: 
3. In the end we might move them to not being run on eveyr commit :D, but currently they are in development :P

Reply:
We can move this to its own job, and have it run only when files in `js` were changed with https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths

That said, if it takes less than 10 minutes, this is probably good enough...